### PR TITLE
New normalization code; bugifx

### PR DESCRIPTION
**Pull request by chazzmoney**

### CODE REVIEW

File: tensortrade/features/scalers/price_normalizer.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+from gym import Space
+from copy import copy
+from typing import Union, List, Tuple, Dict
+
+from tensortrade.features.feature_transformer import FeatureTransformer
+
+
+class PriceNormalizer(FeatureTransformer):
+    """A transformer for normalizing values within a feature pipeline by the column-wise extrema."""
```
Comment (notadamking) [inline tensortrade/features/scalers/price_normalizer.py]: This needs to be updated.

Comment (notadamking) [reply tensortrade/features/scalers/price_normalizer.py]: An explanation of when/why to use this transformer should also be added here.

File: tensortrade/features/scalers/price_normalizer.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import numpy as np
+
+from gym import Space
+from copy import copy
+from typing import Union, List, Tuple, Dict
+
+from tensortrade.features.feature_transformer import FeatureTransformer
+
+
+class PriceNormalizer(FeatureTransformer):
```
Comment (notadamking) [inline tensortrade/features/scalers/price_normalizer.py]: Perhaps this should be renamed to `ComparisonNormalizer` or similar, and `price_column` should be renamed to `comparison_column`, to increase the generalizability.

File: tensortrade/features/scalers/percent_normalizer.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+from gym import Space
+from copy import copy
+from typing import Union, List, Tuple, Dict
+
+from tensortrade.features.feature_transformer import FeatureTransformer
+
+
+class PercentChangeNormalizer(FeatureTransformer):
+    """A transformer for normalizing values within a feature pipeline by the column-wise extrema."""
```
Comment (notadamking) [inline tensortrade/features/scalers/percent_normalizer.py]: This description needs to be updated. An explanation would be nice here too, but is not necessary as this normalizer is more self-explanatory.

File: tensortrade/features/scalers/percent_normalizer.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+    def __init__(self,
+                 columns: Union[List[str], str, None] = None,
+                 price_column: str = 'open',
```
Comment (notadamking) [inline tensortrade/features/scalers/percent_normalizer.py]: This `price_column` arg needs to be removed.

Comment (chazzmoney) [reply tensortrade/features/scalers/percent_normalizer.py]: oops. good catch!

File: tensortrade/features/scalers/percent_normalizer.py (hunk 3 above)
Comment (chazzmoney) [reply tensortrade/features/scalers/percent_normalizer.py]: will do

File: tensortrade/features/scalers/price_normalizer.py (hunk 2 above)
Comment (chazzmoney) [reply tensortrade/features/scalers/price_normalizer.py]: will do

File: tensortrade/features/scalers/price_normalizer.py (hunk 1 above)
Comment (chazzmoney) [reply tensortrade/features/scalers/price_normalizer.py]: will do

**Comment by chazzmoney** [comment]
feeling good. maybe get one more reviewer?

**Comment by chazzmoney** [comment]
not ready, one sec

**Comment by notadamking** [comment]
Had to fix some bugs when I pulled it down. Merging in #118

**Pull request closed without merging**